Sources/Apodini/Semantic Model Builder/Model/Endpoint.swift

```diff
@@ -121,6 +118,18 @@ struct Endpoint<H: Handler>: AnyEndpoint {
     }
 }
 
+extension Endpoint {
+    var description: String {
+        let description = self.context.get(valueFor: DescriptionContextKey.self)
+        return description == DescriptionContextKey.defaultValue ? String(describing: type(of: self.handler)): description
```

Comment:
I am not sure if I understand your comment... the approach was to return the `H.self` type name in case there was no description provided via `DescriptionModifier`.

Reply:
So the idea is that `.description` always returns the struct name (like `"H.self"`).
And the description from the `DescrptionModifier` is stored in a different property, if that makes sense?